src/linker/Linker.Steps/BaseAssemblyStep.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Mono.Cecil;
+
+namespace Mono.Linker.Steps
+{
+
+	public abstract class BaseAssemblyStep : IStep
+	{
+		private LinkContext context;
+
+		protected readonly AssemblyDefinition assembly;
```

Comment:
Code style comment: I personally would prefer the `_assembly` style for fields. We do use it in places in the linker (for example LinkContext is done this way).

I don't know if there's a guide one way or another... so I can't argue either way.

My personal preference is visible difference between fields and local variables.

Reply:
In this specific case I may actually change this to a readonly property... but again, just personal preference.

Reply:
I changed `context` to be a property, and `assembly` to `_assembly` (I didn't make it a property because it looks too much like `System.Reflection.Assembly` which is used by some derived classes). I'm never sure which of the inconsistent conventions to follow - I'm happy to take suggestions.